### ISSUE TITLE
neural network

### ISSUE THREAD

**Issue by Blubberli** [Blubberli/ConllSharedTask#1]
learn how to use tensorflow
which kind of network do we use
word vector issues

**Comment by Blubberli** [comment]
Link for word vectors trained on wikipedia (300 dimensions)
https://github.com/facebookresearch/fastText/blob/master/pretrained-vectors.md

Which network do we use?

**Comment by Blubberli** [comment]
Neue Idee: 
Zusätzlich zu WordEmbeddings, auch POSembeddings benutzen (und evtl. Labelembeddings)

**Comment by Blubberli** [comment]
Hier ein Link, wie man POS embeddings erstellen kann.
https://radimrehurek.com/gensim/models/word2vec.html

In tensorflow:
https://github.com/tensorflow/models/blob/master/swivel/prep.py
mit Hilfe vom svivel algorithm

**Comment by TobiasElssner** [comment]
Hier ist der Link zum Stanford-Paper:
http://cs.stanford.edu/people/danqi/papers/emnlp2014.pdf

**Comment by Blubberli** [comment]
Morphological Embedding:
https://pdfs.semanticscholar.org/3e8c/10a3b0158f15c3e47ffee1478e332cc2bd23.pdf

**Comment by Blubberli** [comment]
skip gramm

**Comment by Blubberli** [comment]
WORD VECTORS:
train morphological, pos embeddings.
use the three types of embeddings as features. we can try how the results change if we use only one of them as a feature or if we concatenate them and use them as one feature.

**Comment by Blubberli** [comment]
which NN do we use?
1) connection weights: feed forward
2) label: recurrent

**Comment by Blubberli** [comment]
Daniels Folien:

http://www.sfs.uni-tuebingen.de/~ddekok/dl4nlp/#schedule
